pkg/apis/apis.go

```diff
@@ -11,6 +13,7 @@ import (
 
 // AddToScheme adds all Resources to the Scheme
 func AddToScheme(s *runtime.Scheme) error {
+	fmt.Println("adding OpenShift specific resources...")
```

Comment:
zombie :-)

Reply:
2hrs ago, I was thinking about having a linter for such statements! 😬

Reply:
removed in 21bb803